docs/lifecycle/export/help.go

```diff
@@ -0,0 +1,15 @@
+package export
+
+var Usage = []string{"rbe [command options] <release bundle name> <release bundle version>"}
+
+func GetDescription() string {
+	return "Export and download a Release Bundle"
+}
+
+func GetArguments() string {
```

Comment:
I believe a better approach might be to provided the target as an optional argument, instead of an option, such as in the download command. 
Then this help should be modified to look similar to the download help.